app/policies/comment_policy.rb

```diff
@@ -0,0 +1,39 @@
+class CommentPolicy < ApplicationPolicy
+  class Scope < Scope
+    # NOTE: Be explicit about which records you allow access to!
+    # def resolve
+    #   scope.all
+    # end
+  end
+
+    def index?
+      true
+    end
+  
+    def show?
+      user == record.author ||
+      !record.author.private? ||
+      record.author.followers.include?(user)
```

Comment:
I'm wondering if this works correctly:
1. I can see my own comments
2. if I - the record author - am not private, then my comments are visible
3. People who follow me can see my comments

This means people who do not follow me, if I'm private, won't see my comments. Is this correct?

Reply:
Just realized that there is no difference of If I leave it with like that or just checking the first condition